manifests/operators/prometheus-cf-exporter.yml

```diff
@@ -0,0 +1,67 @@
+# Should be used in combination with ./cf/firehose-exporter-uaa-client.yml
+
+# Prometheus Scrape Config
+- type: replace
+  path: /instance_groups/name=prometheus/jobs/name=prometheus/properties/prometheus/scrape_configs/-
+  value: 
+    job_name: cf
+    static_configs:
+    - targets: 
+      - localhost:9193 # cf_exporter
+      - localhost:9186 # firehose_exporter
+
+# Exporter jobs
+- type: replace
+  path: /instance_groups/name=prometheus/jobs/-
+  value: 
+    name: cf_exporter
+    release: prometheus
+    properties:
+      cf_exporter:
+        metrics:
+          environment: ((metrics_environment))
+        skip_ssl_verify: ((skip_ssl_verify))
+        cf:
+          api_url: ((cf_api_url))
+          username: ((cf_admin_username))
```

Comment:
Better to use uaa client id and secret, when using user & password, the cf exporter stop working after some time (when the refresh token expires).